network.ts

```diff
@@ -1,23 +1,23 @@
 export interface TablelandNetworkConfig {
```

Comment:
adjusted this interface so it can be reused below—i _don't think_ this should affect anything downstream but calling it out, in case i should create a separate interface for block polling times.